src/main/java/org/benetech/mathshare/converters/UrlCodeConverter.java

```diff
@@ -72,5 +72,6 @@ private static Logger getLogger() {
         return LoggerFactory.getLogger(UrlCodeConverter.class);
     }
 
-    private UrlCodeConverter() { }
+    private UrlCodeConverter() {
```

Comment:
we could use Lombok's @UtilityClass, what do you think?

Reply:
I haven't think about it, thanks.